extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -276,6 +277,17 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
+	async addNotebookToBook(): Promise<void> {
```

Comment:
`this.books` has both books and Notebooks? See previous comment about confusing names, can you clean this up so it matches more closely to what's actually being stored in there?